src/utils/validators.js

```diff
@@ -50,4 +50,34 @@ static async validateOneWayTripRequest(tripRequestData) {
   });
   return schema.validateAsync(tripRequestData, { abortEarly: false });
   }
+
+/**
+* @param {object} tripRequestData one way trip request data
+* @returns {Promise<any>} a Promise of validation output
+*/
+static async validateMultiCitiesTripRequest(tripRequestData) {
+  const { createValidationErrors } = Validators;
+  const joiArray = Joi.array().items(Joi.string()).required();
+  const schema = Joi.object({
+    travelTo: joiArray
+    .messages(createValidationErrors('string', invalidTravelTo)),
+    travelFrom: joiArray
+    .messages(createValidationErrors('string', invalidTravelFrom)),
+    travelReason: Joi.string().required()
+    .messages(createValidationErrors('string', invalidTravelReason)),
+    travelType: Joi.string()
```

Comment:
Could you use an array of pre-defined values, for eg. ["one-way", "return-trip", "multi-city"] as the only valid inputs? Because currently the type can be anything and it doesn't make sense to have a travel type of, let's say "hello"
![Screenshot from 2020-03-23 00-45-06](https://user-images.githubusercontent.com/45935765/77263451-2553ba00-6ca1-11ea-8395-f6db90102822.png)